configs/vision/pathology/offline/segmentation/total_segmentator_2d.yaml

```diff
@@ -135,7 +133,7 @@ data:
             split: test
     dataloaders:
       train:
-        batch_size: &BATCH_SIZE ${oc.env:BATCH_SIZE, 64}
+        batch_size: &BATCH_SIZE ${oc.env:BATCH_SIZE, 16}
```

Comment:
why did the batch size change here?

Reply:
I just copied the configs from `online`

Reply:
if they are contradicting, i would assume the one from offline is currently in use, and online should be adjusted?

Reply:
yes let’s keep the 64

Reply:
done